### PR TITLE
feat: add homepage styles and video

### LINKED ISSUE #6: Homepage and Banner
Audit the homepage use of the MDX components and add the Video banner

### PR DESCRIPTION
**Pull request by alisonjoseph**
Closes #6 

ToDo

- [x] Add responsive images to homepage feature tiles

### CODE REVIEW

**Comment by vpicone** [comment]
@alisonjoseph it looks like the carbon site uses a 2x1 aspect ratio for these cards on mobile. I made a 1x1 square version and opened this issue on the theme to look into other aspect ratios for feature cards https://github.com/carbon-design-system/gatsby-theme-carbon/issues/304

File: src/styles/_homepage.scss [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  }
+
+  .bx--aspect-ratio--2x1 {
+    padding-bottom: 100%;
+
+    @include carbon--breakpoint('md') {
+      padding-bottom: 50%;
+    }
+  }
+}
```
Comment (alisonjoseph) [inline src/styles/_homepage.scss]: @vpicone this was the style override I was doing to get the 1-1 ratio for mobile. agree we need a better way

**Comment by alisonjoseph** [comment]
@vpicone not sure the images are right at mobile?? Here is a screenshot from the live site

Screen Shot 2019-08-02 at 2 03 03 PM

**Comment by alisonjoseph** [comment]
Also I need to update tablet to be 320px tall

File: src/styles/_homepage.scss (hunk 1 above)
Comment (vpicone) [reply src/styles/_homepage.scss]: ahhh I see, yeah I think that'd crop the developer one

**Comment by vpicone** [comment]
@alisonjoseph I made square versions of illustrations until we can add aspect ratio support for the feature card in the theme (or figure out a hack here)

**Comment by alisonjoseph** [comment]
maybe my hack is messing up your square images, let me take a closer look

**Pull request merged**